samples/MeteoControl/app/application.cpp

```diff
@@ -54,7 +54,7 @@ void init()
 	lcd.createChar(6, icon_cross);
 	lcd.createChar(7, icon_check);
 
-	WifiStation.config(ActiveConfig.NetworkSSID, ActiveConfig.NetworkPassword);
+	WifiStation.config(ActiveConfig.NetworkSSID.c_str(), ActiveConfig.NetworkPassword.c_str());
```

Comment:
> Also there should be quantitative measurement that proves that this effort is worth it.

As you probably know, there's no way to measure heap fragmentation on ESP. At least not yet.